.github/workflows/main.yml

```diff
@@ -130,7 +130,8 @@ jobs:
           sudo apt-get update
           sudo apt-get install build-essential gfortran libopenblas-dev
       - name: Setup Python
-        uses: actions/setup-python@v2
+        id: python-install
+        uses: actions/setup-python@v4
```

Comment:
Updating this version is a hard prerequisite because the `python-version` output is only provided since v4.